packages/cli/src/commands/cms-sync.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from '@oclif/core'
+import { spawn } from 'child_process'
+import { existsSync } from 'fs-extra'
+import { tmpDir } from '../utils/directory'
+
+export default class CmsSync extends Command {
+  async run() {
+    if (!existsSync(tmpDir)) {
+      throw Error(
+        'The ".faststore" directory could not be found. If you are trying to serve your store, run "faststore build" first.'
```

Comment:
This error message is misleading. No one is trying to serve their store when running `faststore cms-sync`.

Reply:
I'd just say that to run `faststore cms-sync` either a `yarn build` or a `yarn dev` will have to be executed first. Alternatively, can't we just "generate" the .faststore folder at this moment with what's currently on their directory? I believe we have a function for doing just that.